packages/bcer-retailer-app/app/src/views/productReport/DeleteProductSubmission.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { makeStyles, Typography, Paper } from '@material-ui/core';
+import { CSVLink } from 'react-csv';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import ChatBubbleOutlineIcon from '@material-ui/icons/ChatBubbleOutline';
+import SaveAltIcon from '@material-ui/icons/SaveAlt'
+import moment from 'moment';
+
+import { StyledTable, StyledButton, StyledConfirmDialog } from 'vaping-regulation-shared-components';
+import { Products } from '@/constants/localInterfaces';
+import { ProductReportHeaders } from '@/constants/localEnums';
+import { useAxiosDelete, useAxiosGet } from '@/hooks/axios';
+
+const useStyles = makeStyles({
+  buttonIcon: {
+    paddingRight: '5px',
+    color: '#285CBC',
+  },
+  title: {
+    padding: '20px 0px',
+    color: '#002C71'
+  },
+  helpTextWrapper: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: '20px',
+    backgroundColor: '#E0E8F0',
+    marginBottom: '30px',
+    borderRadius: '5px',
+  },
+  helperIcon: {
+    fontSize: '45px',
+    color: '#0053A4',
+    paddingRight: '25px',
+  },
+  box: {
+    border: 'solid 1px #CDCED2',
+    borderRadius: '4px',
+    padding: '1.4rem',
+  },
+  boxTitle: {
+    paddingBottom: '10px'
+  },
+  tableRowCount: {
+    paddingBottom: '10px'
+  },
+  actionsWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    paddingBottom: '10px'
+  },
+  csvLink: {
+    textDecoration: 'none',
+  },
+  submitWrapper: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    paddingTop: '30px'
+  },
+  checkboxLabel: {
+    marginTop: '20px'
+  },
+});
+
+export default function DeleteProductSubmissions() {
+  const classes = useStyles();
+  const history = useHistory();
+  const [submissionDate, setSubmissionDate] = useState<string>();
+  const [deleteConfirmOpen, setDeleteConfirmOpen] = useState<boolean>(false);
+  const { submissionId } : { submissionId: string } = useParams();
+
+  const [{ data: products = [] }] = useAxiosGet(`/products?submissionId=${submissionId}`);
+  const [{}, request] = useAxiosDelete(`/products/submission/${submissionId}`, { manual: true });
+
+  useEffect(() => {
+    if (!!products.length) {
+      setSubmissionDate(moment(products[0].created_at).format('LLL'));
+    }
+  }, [products]);
+
+  
+  const confirmDelete = async() => {
+    await request();
+    history.push('/products');
```

Comment:
this will navigate to /products even if the delete request fails. is that intentional?

Reply:
Good point!